dll/ntdll/ldr/ldrapi.c

```diff
@@ -309,10 +309,11 @@ LdrLockLoaderLock(
 NTSTATUS
 NTAPI
 DECLSPEC_HOTPATCH
-LdrLoadDll(IN PWSTR SearchPath OPTIONAL,
-           IN PULONG DllCharacteristics OPTIONAL,
-           IN PUNICODE_STRING DllName,
-           OUT PVOID *BaseAddress)
+LdrLoadDll(
+    _In_opt_ PWSTR SearchPath,
+    _In_opt_ PULONG DllCharacteristics,
+    _In_ PUNICODE_STRING DllName,
+    _Out_ PVOID* BaseAddress)
```

Comment:
```suggestion
    _Out_ PVOID *BaseAddress)
```

Reply:
Committed.

Reply:
That change was not really necessary (imho putting the asterisk next to the variable name, instead of together with the type, is not really logical, since the type of BaseAddress is "PVOID*" )

Reply:
Committed.